packages/commonwealth/client/scripts/views/pages/CommunityManagement/Topics/CreateTopicsSection/validation.ts

```diff
@@ -0,0 +1,12 @@
+import z from 'zod';
+import { VALIDATION_MESSAGES } from '../../../../../helpers/formValidationMessages';
+VALIDATION_MESSAGES;
+
+export const topicCreationValidationSchema = z.object({
+  topicName: z
+    .string({ invalid_type_error: VALIDATION_MESSAGES.NO_INPUT })
+    .min(1),
```

Comment:
I think we should add a message here if the min condition is not satisfied, otherwise, it will give the default error (which is also fine), we already have a form validation message for max chars limit tho
```
MAX_CHAR_LIMIT_REACHED: 'Max character limit reached',
```

<img width="330" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/51641047/08696b68-2218-4e24-b7cf-0775048fc658">


```diff
+ min(1, { message: `{message here}` })
```

Plz update if you think its best, else we can resolve this.

Reply:
As far as I can tell, there is no maximum character limit imposed anywhere on topic name length. As such, I don't think it's proper to add one. That task, if it happens, would be best left to another PR.

Reply:
we should use     `min(1, VALIDATION_MESSAGES.NO_INPUT)` instead of default error `string must contain at least 1 character(s)`

Reply:
Done!